traffic_portal/app/src/common/modules/form/serverCapability/form.serverCapability.tpl.html

```diff
@@ -41,6 +41,13 @@
                     <span ng-show="hasError(serverCapabilityForm.name)" class="form-control-feedback"><i class="fa fa-times"></i></span>
                 </div>
             </div>
+            <div class="form-group" ng-class="{'has-error': hasError(serverCapabilityForm.description), 'has-feedback': hasError(serverCapabilityForm.description)}">
```

Comment:
I don't think it's possible for this field to have errors or feedback

Reply:
ok